internal/planner/commit.go

```diff
@@ -142,13 +144,14 @@ func (n *dagScanNode) simpleExplain() (map[string]any, error) {
 
 	// Build the explanation of the spans attribute.
 	spansExplainer := []map[string]any{}
+	undefinedHsKey := keys.HeadStoreKey{}
```

Comment:
nitpick:
```suggestion
	undefinedHSKey := keys.HeadStoreKey{}
```

Reply:
Headstore is one word, the current capitalization of `HeadStoreKey` is incorrect and has been fixed in the branchable collections branch (is bundled with some other branchable-specific changes, which is why it isnt in this PR).